spring-cloud-sleuth-zipkin/src/main/java/org/springframework/cloud/sleuth/zipkin/ZipkinAutoConfiguration.java

```diff
@@ -31,25 +28,21 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
-import com.github.kristofa.brave.SpanCollector;
 
 /**
  * @author Spencer Gibb
  */
 @Configuration
 @EnableConfigurationProperties
-@ConditionalOnClass(SpanCollector.class)
 @ConditionalOnProperty(value = "spring.zipkin.enabled", matchIfMissing = true)
 public class ZipkinAutoConfiguration {
 
 	@Bean
-	@ConditionalOnMissingBean(SpanCollector.class)
-	public SpanCollector spanCollector() {
+	@ConditionalOnMissingBean(ZipkinSpanReporter.class)
+	public ZipkinSpanReporter reporter() {
 		ZipkinProperties zipkin = zipkinProperties();
 		String url = "http://" + zipkin.getHost() + ":" + zipkin.getPort();
```

Comment:
Is HTTP common for this? I'd expect at least the option to use HTTPS (even if it's not the default)?